test/jest.setup.js

```diff
@@ -0,0 +1,3 @@
+// broadcast-channel should not detect node environment
+// https://github.com/pubkey/broadcast-channel/blob/master/src/util.js#L61
```

Comment:
To fix error `No useable method found in ["native","idb","localstorage"]`
Which happens because broadcast-channel detects node environment:
https://github.com/pubkey/broadcast-channel/blob/master/src/util.js#L61